readthedocs/vcs_support/base.py

```diff
@@ -113,6 +70,15 @@ def make_clean_working_dir(self):
         shutil.rmtree(self.working_dir, ignore_errors=True)
         self.check_working_dir()
 
+    @property
+    def env(self):
+        environment = os.environ.copy()
+
+        # TODO: kind of a hack
+        del environment['PATH']
```

Comment:
I need to delete this one because there is an `assert` in the code that doesn't allow the `PATH` in the environment

Reply:
Probably best anyways? I think we want tight control over this.